specification/sections/01-flag-evaluation.md

```diff
@@ -189,33 +189,45 @@ FlagEvaluationDetails<MyStruct> myStructDetails = client.getObjectDetails<MyStru
 
 ##### Requirement 1.4.7
 
+> If the `flag metadata` field in the `flag resolution` structure returned by the configured `provider` is set, the `evaluation details` structure's `flag metadata` field **MUST** contain that value. Otherwise, it **MUST** contain an empty record.
```

Comment:
Could you add a non-normative section with some pseudo code and a small blurb on the intent behind flag metadata?

Reply:
I agree it does, but I'd prefer if the spec was as self-contained as possible.

An alternative could be adding a definition to `types.md`.

Reply:
Sure, you can see an example [here](https://github.com/open-feature/spec/blob/main/specification/sections/01-flag-evaluation.md#requirement-116). It's just text below the quoted text.